classes/Database/Install.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Ps_Googleanalytics\Database;
+
+class Install
+{
+    /**
+     * @var \Ps_Googleanalytics
+     */
+    private $module;
+
+    public function __construct(\Ps_Googleanalytics $module)
+    {
+        if (\Shop::isFeatureActive()) {
+            \Shop::setContext(\Shop::CONTEXT_ALL);
+        }
+
+        $this->module = $module;
+    }
+
+    /**
+     * installTables
+     *
+     * @return bool
+     */
+    public function installTables()
+    {
+        $sql = [];
+
+        $sql[] = 'CREATE TABLE IF NOT EXISTS `' . _DB_PREFIX_ . 'ganalytics` (
+            `id_google_analytics` int(11) NOT NULL AUTO_INCREMENT,
+            `id_order` int(11) NOT NULL,
+            `id_customer` int(10) NOT NULL,
+            `id_shop` int(11) NOT NULL,
+            `sent` tinyint(1) DEFAULT NULL,
+            `date_add` datetime DEFAULT NULL,
+            PRIMARY KEY (`id_google_analytics`),
+            KEY `id_order` (`id_order`),
+            KEY `sent` (`sent`)
+        ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=utf8 AUTO_INCREMENT=1';
+
+        $sql[] = 'CREATE TABLE IF NOT EXISTS `' . _DB_PREFIX_ . 'ganalytics_data` (
+            `id_cart` int(11) NOT NULL,
+            `id_shop` int(11) NOT NULL,
+            `data` TEXT DEFAULT NULL,
+            PRIMARY KEY (`id_cart`)
+        ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=utf8';
```

Comment:
```suggestion
        ) ENGINE=' . _MYSQL_ENGINE_ . ' DEFAULT CHARSET=utf8_mb4';
```

Ping @PierreRambaud

Reply:
Is there a constant for that ?
Like `_MYSQL_ENGINE_` for example ?
Or another parameters about configuration to use for create table ?

Reply:
Note when we add this change, we have to handle the upgrade as well.

Reply:
After discuting with @PierreRambaud, I think my idea is not so well because, the module has a large compliance version (1.6) and utf8mb4 could generate bugs.